app/views/tpi/home/_testimonials.html.erb

```diff
@@ -1,21 +1,75 @@
 <div class="container">
   <div class="columns">
-    <div class="column is-2 is-offset-1">
-      “
-    </div>
-    <div class="column is-8 is-offset-1">
-      pic
-      <img src="" class="testimonials__avatar">
-
-      <p class="testimonials__content">
-        Businesses should be able explain to investors how they plan to manage climate change risks, invest and innovate on the way to the zero-carbon economy of the future. With the launch of the Transition Pathway Initiative, asset owners from around the world are sending a strong signal that portfolios will align in the future with companies that are taking the transition to a low carbon economy seriously.
-      </p>
-
-      <p class="testimonials__author">
-        Emma Howard Boyd
-        Chair of the Environment Agency
-      </p>
-    </div>
+    <div class="column is-10 is-offset-1 testimonials">
```

Comment:
once you're gonna start adding the behavior to it you might end up moving everything to `TestimonialSlider` component, I think it's gonna be easier than exposing only some functions from the component, don't even know how that would work 🤔

Reply:
Yeah, that's the idea (the other testimonials are not even updated as the 1st one, they moer liek placeholders)...
Right now i'm adding dummy data like:
```
[{
  author: 'Lee Van Cleef',
  role: 'Golden Boot Award Winner',
  content: ['fooo', 'baaar', 'baaaz'],
  avatar_url: 'xxx.jpg',
}]
```

But i have mixed feelings if it is worth throwing react into this...

What do you suggest?

Reply:
I think it's gonna be more natural with React but as you prefer! what did you have in mind in the beginning?

Reply:
a bunch of `addEventListener`s and `querySelector`s replacing `innerHTML`s...